fatiando/mesher.py

```diff
@@ -1720,3 +1720,433 @@ def keep(cell):
             return True
         return False
     return [c for c in cells if keep(c)]
+
+
+class TriaxialEllipsoid (GeometricElement):
+
+    """
+    Create a triaxial ellipsoid.
+
+    .. note:: The coordinate system used is x -> North, y -> East
+    and z -> Down
+
+    Parameters:
+
+    * x, y, z : float
+        The coordinates of the center of the ellipsoid
+    * a, b, c : float
+        Semi-axes of the ellipsoid (a>b>c)
+    * alpha, gamma, delta
+        Orientation angles of the ellipsoid
+    * props : dict
+        Physical properties assigned to the ellipsoid.
+        Ex: ``props={'density':10,
+                     'remanence':[10000., 25., 40.],
+                     'k':[0.562, 0.485, 0.25, 90., 0., 0.]}``
```

Comment:
It is complicated task. `k` is the susceptibility tensor. I have replaced `k` by `susceptibility tensor`. The susceptibility tensor is defined as follows (in Latex):
$$
\mathbf{K} = \mathbf{U} \left[
\begin{array}{ccc}
k_{1} & & \\
& k_{2} & \\
& & k_{3} \\
\end{array}
\right] \mathbf{U}^{\top} \: ,
$$
where $k_{1}$, $k_{2}$ and $k_{3}$ are the *principal susceptibilities* and $\mathbf{U}$ is an orthogonal matrix whose columns are defined by unit vectors $\mathbf{u}_{1}$, $\mathbf{u}_{2}$ and $\mathbf{u}_{3}$. These vectors are called *principal directions*. We define the principal directions by using three angles: strike, rake and dip. These angles have the same geometrical meaning as those used to define the spatial orientation of the ellipsoid, but they are different.

We define the susceptibility tensor as a list containing the principal susceptibilities and the three angles. I am not sure if this is the best way of defining the susceptibility. This way can be used to define any (I am not sure) matrix. Notice that the principal susceptibilities represent the eigenvalues and $\mathbf{U}$ is the matrix containing the eigenvectors of the susceptibility tensor. Maybe, we could create a function similar to the `utils.ang2vec`. This function would receive the eigenvalues and three angles defining the eigenvectors and return the resultant matrix. What do you think?